docs/configuration.asciidoc

```diff
@@ -671,6 +671,34 @@ the agent will capture request and response headers, including cookies.
 
 NOTE: Setting this to `false` reduces memory allocations, network bandwidth and disk space used by Elasticsearch.
 
+[float]
+[[config-transaction-ignore-urls]]
+==== `TransactionIgnoreUrls` (performance)
+
+[options="header"]
+|============
+| Environment variable name             | IConfiguration or Web.config key
+| `ELASTIC_APM_TRANSACTION_IGNORE_URLS` | `ElasticApm:TransactionIgnoreUrls`
+|============
+
+[options="header"]
+|============
+| Default                                                                                                                    | Type
+| `/VAADIN/*, /heartbeat*, /favicon.ico", *.js", *.css", *.jpg", *.jpeg", *.png", *.gif", *.webp", *.svg", *.woff", *.woff2` | List<string>
+|============
+
+Used to restrict requests to certain URLs from being instrumented.
+
+This property should be set to a list containing one or more strings.
+When an incoming HTTP request is detected, its request path will be tested against each element in this list.
```

Comment:
@bmorelli25 I think this part could be improved a lot. In the issue description I write about what part of the URL is matched, but I think this text does not make it clear.

Can you come up with something better? I'd like to have something that describes very well what part of the URL is exactly matched - I think most docs just call this part 'path`, but I'm not sure its meaning is generally known.

Reply:
You know, I actually don't find this difficult to understand. With that being said, an example here certainly wouldn't hurt and would go a long way to making this feature easier to understand. What about adding what you have in the description to these docs?

```
Used to ...

This property should be set to a list containing one or more strings.
When an incoming HTTP request is detected, its request path will be tested against each element in this list.
For example, adding `/home/index` to this list would match and remove instrumentation from the following URLs:

[source,txt]
----
https://www.mycoolsite.com/home/index
http://localhost/home/index
http://whatever.com/home/index?value1=123
----

This option supports ...
```

Reply:
Great; I also think adding an example helps. I added it. Thanks!